warehouse/utils/utils.go

```diff
@@ -934,3 +937,27 @@ func GetLoadFilePrefix(timeWindow time.Time, warehouse WarehouseT) (timeWindowFo
 	}
 	return timeWindowFormat
 }
+
+func GetRequestWithTimeout(url string, timeout time.Duration) ([]byte, int, error) {
```

Comment:
Added context. From @abhimanyubabbar

Reply:
Use context.TODO() for now. But add a comment to change this later. Use application context going forward.

Reply:
Done.